requirements-dev.txt

```diff
@@ -11,7 +11,7 @@ cpplint
 flake8<3.8.0
 flake8-comprehensions>=3.1.0
 flake8-rst>=0.6.0,<=0.7.0
-isort==4.3.21
+isort>=5.2.1
```

Comment:
are the changes backwards compatible or is this the minimum version we would need

Reply:
They fixed the conflict with MyPy check (#35134) in either version 5.2.0 or 5.2.1. So I just set it to the latest one.